paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/CardDetailsUI.kt

```diff
@@ -131,15 +129,15 @@ private fun CardDetailsEditUI(
 
 @Composable
 private fun CardNumberField(
-    card: PaymentMethod.Card,
+    last4: String?,
     selectedBrand: CardBrandChoice,
     availableNetworks: List<CardBrandChoice>,
     shouldShowCardBrandDropdown: Boolean,
     savedPaymentMethodIcon: Int,
     onBrandChoiceChanged: (CardBrandChoice) -> Unit,
 ) {
     CommonTextField(
-        value = "•••• •••• •••• ${card.last4}",
+        value = "•••• •••• •••• $last4",
```

Comment:
nit - if last4 can be null, should we add a last set of ••••?

Reply:
This shouldn't happen for payment methods coming from the backend, but I can add `${last4 ?: "••••"}`.